test/RequestQueueTests.js

```diff
@@ -1,300 +1,290 @@
-				.useQueue()
-				.end( function( err, res ) {
-					failureCount++;
-					assert404andFailureCount( failureCount, 1, err.status );
-				});
-
-			superagent
-				.get( 'http://localhost:5000/faiure' )
-				.useQueue()
-				.end( function( err, res ) {
-					failureCount++;
-					assert404andFailureCount( failureCount, 2, err.status );
-				});
-
-			superagent
-				.get( 'http://localhost:5000/faiure' )
-				.useQueue()
-				.end( function( err, res ) {
-					failureCount++;
-					assert404andFailureCount( failureCount, 3, err.status );
-				});
-
-			superagent
-				.get( 'http://localhost:5000/faiure' )
-				.useQueue()
-				.end( function( err, res ) {
-					failureCount++;
-					assert404andFailureCount( failureCount, 4, err.status );
-					done();
-				});
-		});
-
-		it( 'use queue, request timedout, expect successful retry', function( done ) {
-
-			this.timeout( 10000 );
-
-			var retryHandler = function( err ) {
-				err.code.should.equal( TIMEOUT_RESPONSE );
-			}
-
-			superagent
-				.get( 'http://localhost:5000/timeout' )
-				.useQueue()
-				.timeout( 100 )
-				.retryOnConnectionFailure( retryHandler )
-				.end( function( err, res ) {
-					res.text.should.equal( SUCCESS_RESPONSE );
-					done();
-				});
-		})
-
-
-		it( 'no queue, request timedout, expect successful retry', function( done ) {
-
-			this.timeout( 10000 );
-
-			var retryHandler = function( err ) {
-				err.code.should.equal( TIMEOUT_RESPONSE );
-			}
-
-			superagent
-				.get( 'http://localhost:5000/timeout' )
-				.timeout( 100 )
-				.retryOnConnectionFailure( retryHandler )
-				.end( function( err, res ) {
-					res.text.should.equal( SUCCESS_RESPONSE );
-					done();
-				});
-		})
-
-		it( 'use queue, gateway error, expect successful retry', function( done ) {
-
-			this.timeout( 10000 );
-
-			var retryHandler = function( err ) {
-				err.status.should.equal( 503 );
-
-			}
-
-			superagent
-				.get( 'http://localhost:5000/gatewayFailure' )
-				.useQueue()
-				.timeout( 100 )
-				.retryOnConnectionFailure( retryHandler )
-				.end( function( err, res ) {
-					res.text.should.equal( SUCCESS_RESPONSE );
-					done();
-				});
-		})
-
-		function assert404andFailureCount( failureCount, expectedFailureCount, status ) {
-			failureCount.should.equal( expectedFailureCount );
-			status.should.equal( 404 );
-		}
-
-		function assertSuccessAndRequestsLeft( requestsLeft, expectedRequestsLeft, response ) {
-			requestsLeft.should.equal( expectedRequestsLeft );
-			response.should.equal( SUCCESS_RESPONSE );
-		}
-
-		function getConcurrencyChecker( expectedConcurrency ) {
-			return setInterval( function() {
-				concurrentRequests.should.equal( expectedConcurrency );
-			}, 200 );
-		}
-
-		function closeTestIfNoRequests( requestsLeft, done, concurrencyChecker ) {
-			if ( requestsLeft === 0 ) {
-				done();
-				clearInterval( concurrencyChecker );
-			}
-		}
-	});
-});
\ No newline at end of file
+'use strict';
```

Comment:
strict mode needs to be removed for const to be used in the tests. TravisCI is failing on tests https://travis-ci.org/Brightspace/superagent-d2l-queue/builds/117468921